stups-spring-oauth2-server/src/main/java/org/zalando/stups/oauth2/spring/server/DefaultTokenInfoRequestExecutor.java

```diff
@@ -73,9 +76,38 @@ public DefaultTokenInfoRequestExecutor(final String tokenInfoEndpointUrl, RestOp
     }
     //@formatter:on
 
+    /**
+     * Creates a {@link RestTemplate} instance with a default
+     * {@link TokenInfoResponseErrorHandler}.
+     * 
+     * @see TokenInfoResponseErrorHandler#getDefault()
+     */
     public static RestTemplate buildRestTemplate() {
+        return buildRestTemplate(TokenInfoResponseErrorHandler.getDefault());
+    }
+
+    /**
+     * Creates a {@link RestTemplate} instance with a
+     * {@link TokenInfoResponseErrorHandler} that will not do any
+     * {@link ResponseErrorHandler#handleError(org.springframework.http.client.ClientHttpResponse)}
+     * for the passed {@link HttpStatus}.
+     * 
+     * @param unhandledStatusSet
+     */
+    public static RestTemplate buildRestTemplate(EnumSet<HttpStatus> unhandledStatusSet) {
+        return buildRestTemplate(new TokenInfoResponseErrorHandler(unhandledStatusSet));
```

Comment:
If this line would be covered by a unit test, you complete PR diff would have :100:% coverage :smile: